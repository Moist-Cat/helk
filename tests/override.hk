type Point {
    x = "0";
    y = 0;

    do(x) => self.x;
};

type Point2 inherits Point{
    z = 0;

    do(x) => self.z + x;
};

let a = new Point("doko", 1);
let b = new Point2("1", 1, 10);
prints(a.do(1));
print(b.do(3));
