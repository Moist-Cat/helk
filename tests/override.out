doko
13.000000
